### DISCUSSION TITLE
请问一下，为啥接入公众号回复的速度那么慢呀？

### DISCUSSION THREAD

**Discussion by LPtrichor** [zhayujie/chatgpt-on-wechat#1022] [Q&A]
请问一下，为啥接入公众号回复的速度那么慢呀？有没有啥解决办法呢

**Comment by Pluto-YSZBD** [comment]
想请教一下你，你接入到公众号不会被封号的嘛

**Comment by LPtrichor** [comment]
不会，接入微信号可能会被封



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月6日(周四) 晚上7:47
收件人: ***@***.***&gt;;
抄送: "Luo ***@***.******@***.***&gt;;
主题: Re: [zhayujie/chatgpt-on-wechat] 请问一下，为啥接入公众号回复的速度那么慢呀？ (Discussion #1022)




 
想请教一下你，你接入到公众号不会被封号的嘛

**Reply by Pluto-YSZBD** [reply]
你接入公众号多久了?使用的人多不多？我接入微信已经快三个月了没有封号嘞，我觉得很稳鸭

**Reply by Pluto-YSZBD** [reply]
你微信多少，可以加个微信交流一下不

**Comment by LPtrichor** [comment]
可以
v:jasper_1888




---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月6日(周四) 晚上7:50
收件人: ***@***.***&gt;;
抄送: "Luo ***@***.******@***.***&gt;;
主题: Re: [zhayujie/chatgpt-on-wechat] 请问一下，为啥接入公众号回复的速度那么慢呀？ (Discussion #1022)




 
你微信多少，可以加个微信交流一下不
